resources/faq.md

```diff
@@ -55,7 +55,7 @@ It means that in many cases (e.g., documents addition), you will receive as serv
 This kind of **successful response** indicates that the operation has been taken into account, but may not have been executed yet.<br>
 
 ::: tip
-You can check the status of the operation using this `updateId` and the [get update status route](/references/updates.md#get-an-update-status).<br>
+You can check the status of the operation via the `updateId` and the [get update status route](/references/updates.md#get-an-update-status).<br>
```

Comment:
Via is more a way to get somewhere than a tool to do something. https://dictionary.cambridge.org/fr/dictionnaire/anglais/via
```suggestion
You can check the status of the operation using the `updateId` via the [get update status route](/references/updates.md#get-an-update-status).<br>
```

Reply:
It makes more sense thanks!